flipperdevices/flipperzero-firmware
# Storage: lfs config fingerprinting. RTC: fix data collision in lock register, refactor and cleanup.

# What's new

- Storage: lfs config fingerprinting.
- RTC: fix data collision in lock register, refactor and cleanup.
- Fix `DEBUG=0 COMPACT=1` build

# Verification 

- Compile and upload
- Check locking with pin code
- LFS now forcedly formatted even if superblock is present and looks valid, but geometry mismatch
- Check RTC reg functions: reboot, reboot to dfu

# Checklist (do not modify)
